ecs_admin.tf

```diff
@@ -0,0 +1,217 @@
+# #########################################
+# ECS General Resources
+# #########################################
+data "aws_iam_policy_document" "admin_ecs_assume_role_policy" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role" "admin_ecs_task_execution" {
+  name               = "${module.labels.id}-admin-task-exec-role"
+  assume_role_policy = data.aws_iam_policy_document.admin_ecs_assume_role_policy.json
+}
+
+resource "aws_iam_role_policy_attachment" "admin_ecs_task_execution" {
+  role       = aws_iam_role.admin_ecs_task_execution.name
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
+}
+
+resource "aws_iam_role" "admin_ecs_task_role" {
+  name               = "${module.labels.id}-admin-task-role"
+  assume_role_policy = data.aws_iam_policy_document.admin_ecs_assume_role_policy.json
+}
+
+data "aws_iam_policy_document" "admin_ecs_task_policy" {
+  statement {
+    actions = ["ssm:GetParameter", "ssm:GetParameters"]
+    resources = concat([
+      aws_ssm_parameter.admin_cognito_user_pool_id.arn,
+      aws_ssm_parameter.admin_cognito_region.arn,
+      aws_ssm_parameter.admin_cors_origin.arn,
+      aws_ssm_parameter.admin_host.arn,
+      aws_ssm_parameter.admin_port.arn,
+      aws_ssm_parameter.cors_origin.arn,
+      aws_ssm_parameter.db_database.arn,
+      aws_ssm_parameter.db_host.arn,
+      aws_ssm_parameter.db_pool_size.arn,
+      aws_ssm_parameter.db_port.arn,
+      aws_ssm_parameter.db_reader_host.arn,
+      aws_ssm_parameter.db_ssl.arn,
+      aws_ssm_parameter.log_level.arn,
+      aws_ssm_parameter.push_service_url.arn,
+      aws_ssm_parameter.settings_lambda.arn,
+      ],
+      aws_ssm_parameter.security_callback_rate_limit_request_count.*.arn,
+    aws_ssm_parameter.security_callback_rate_limit_secs.*.arn)
+  }
+
+  statement {
+    actions = ["secretsmanager:GetSecretValue"]
+    resources = [
+      data.aws_secretsmanager_secret_version.rds_read_only.arn,
+      data.aws_secretsmanager_secret_version.rds_read_write.arn,
+      data.aws_secretsmanager_secret_version.admin_push_service_token.arn,
+      data.aws_secretsmanager_secret_version.google_maps_api_key.arn
+    ]
+  }
+
+  statement {
+    actions = ["kms:*"]
```

Comment:
Is it possible to remove the `*` and target to the specific required actions?

Reply:
Removed as not used